src/components/ChangePlan/PlanCalculator.test.js

```diff
@@ -67,6 +69,16 @@ describe('PlanCalculator component', () => {
       expect(getByText('plan_calculator.plan_type_prepaid')).toBeInTheDocument();
       expect(getByText('plan_calculator.plan_type_subscribers')).toBeInTheDocument();
       expect(getByText('plan_calculator.plan_type_monthly_deliveries')).toBeInTheDocument();
+
+      // check the order of the plan types
+      const navigatorTabs = container.querySelector('.tabs-nav');
+      const it = planTypes[Symbol.iterator]();
+      let tab = navigatorTabs.firstChild;
+      while (tab) {
+        const planType = it.next().value;
+        expect(tab).toHaveTextContent(`plan_calculator.plan_type_${planType.replace('-', '_')}`);
+        tab = tab.nextSibling;
+      }
```

Comment:
Is there not a high-level way of doing this? (without using `Symbol`, `next()`, and `while`)

Reply:
Yes. I don't know why I did it like that.It's better that way: 
```
planTypes.forEach((planType) => {
    expect(tab).toHaveTextContent(`plan_calculator.plan_type_${planType.replace('-', '_')}`);
    tab = tab.nextSibling;
});
```